otelfiber/fiber.go

```diff
@@ -16,7 +19,7 @@ import (
 
 const (
 	tracerKey  = "otel-go-contrib-tracer-gofiber-fiber"
```

Comment:
I think you should update tracerKey also.

Reply:
@efectn what is your suggestion? I am borrowing from https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/github.com/labstack/echo/otelecho/echo.go#L32

Reply:
fiber-contrib-tracer-gofiber-fiber 🤔